src/plugins/discover/public/application/main/components/sidebar/discover_sidebar.tsx

```diff
@@ -412,7 +413,7 @@ export function DiscoverSidebarComponent({
                   initialIsOpen={true}
                   buttonContent={
                     <EuiText size="xs" id="available_fields">
-                      <strong>
+                      <strong id={DISCOVER_TOUR_STEP_ANCHOR_IDS.addFields}>
```

Comment:
Just loud thinking ... wonder why there is no `.eui-textStrong`  we could use for <EuiText, then the `<strong>` would be redundant, the could be assigned to EuiText, `available_fields` seems to be unused anyway ... however.. just loud thinking :)